djvm/src/main/kotlin/net/corda/djvm/rules/implementation/ReturnTypeWrapper.kt

```diff
@@ -41,9 +41,9 @@ object ReturnTypeWrapper : Emitter {
     override fun emit(context: EmitterContext, instruction: Instruction) = context.emit {
         if (instruction is MemberAccessInstruction && context.whitelist.matches(instruction.reference)) {
             fun invokeMethod() = when (instruction.operation) {
-                INVOKEVIRTUAL -> invokeVirtual(instruction.owner, instruction.memberName, instruction.descriptor)
-                INVOKESTATIC -> invokeStatic(instruction.owner, instruction.memberName, instruction.descriptor)
-                INVOKESPECIAL -> invokeSpecial(instruction.owner, instruction.memberName, instruction.descriptor)
+                INVOKEVIRTUAL -> invokeVirtual(instruction.className, instruction.memberName, instruction.descriptor)
```

Comment:
What is the difference between `owner` and `className`, is this just a renaming?

Reply:
It _is_ a renaming, but one that allows `MemberAccessInstruction` also to implement `MemberInformation`.